src/fontra/client/core/utils.js

```diff
@@ -221,3 +221,15 @@ export function makeUPlusStringFromCodePoint(codePoint) {
     ? "U+" + codePoint.toString(16).toUpperCase().padStart(4, "0")
     : "";
 }
+
+export async function writeToClipboard(clipboardObject) {
+  if (!clipboardObject) return;
+
+  for (const [key, value] of Object.entries(clipboardObject)) {
+    clipboardObject[key] = new Blob([`${value}`], {
```

Comment:
It is not nice to modify an object we got passed as an argument: it would be better to make a fresh object, something like this (untested):

```js
    const clipboardObject = Object.fromEntries(
      Object.entries(clipboardObject).map((item) => [item[0], new Blob([item[1]])])
    );
```
```